api/graphql/makeModels.js

```diff
@@ -58,6 +58,7 @@ export default function makeModels (userId, isAdmin) {
       filter: nonAdminFilter(q => {
         q.where('users.id', 'in', Membership.query().select('user_id')
           .where('community_id', 'in', myCommunityIds()))
+          .where('communities_users.community_id', 'in', myCommunityIds())
```

Comment:
What I meant is that you want this line to go in a new `filter` prop in the `communities_users` section of the models file, not the `users` section. (In the new arrangement as of #242, that's the Membership section, not the Person section.) What we're trying to do is filter the `communities_users` rows that will be returned for a given user, not filter users based on their community membership. (We're already doing that in the line above, so putting this here is redundant.)